neptune/new/internal/backends/hosted_client.py

```diff
@@ -76,9 +76,6 @@ def _get_token_client(
         endpoint_url: str = None
 ) -> SwaggerClient:
     config_api_url = credentials.api_url_opt or credentials.token_origin_address
-    if proxies is None and endpoint_url is None:
-        verify_host_resolution(config_api_url)
```

Comment:
Wyrzuciłeś to nie w tym miejscu, do którego dałem komentarz. Sprawdzenie tego ma sens wyłącznie przed jakikolwiek pierwszym requestem.